SocialCareCaseViewerApi.Tests/V1/UseCase/AllocationsUseCaseTests.cs

```diff
@@ -78,12 +78,97 @@ public void ListAllocationsByWorkerEmail()
             var request = new ListAllocationsRequest() { WorkerEmail = "test@example.com" };
             var gatewayResponse = new List<Allocation> { new Allocation() { AllocatedWorker = "Test Worker" } };
 
-            _mockDatabaseGateway.Setup(x => x.SelectAllocations(0, 0, "test@example.com"))
+            _mockDatabaseGateway.Setup(x => x.SelectAllocations(0, 0, "test@example.com", 0))
                 .Returns(gatewayResponse);
 
             var response = _allocationsUseCase.Execute(request);
 
             response.Allocations.Should().BeEquivalentTo(gatewayResponse);
         }
+
+        [Test]
+        public void ListAllocationsByTeamIdReturnsExpectedList()
+        {
+            var request = new ListAllocationsRequest() { TeamId = 10 };
+            var gatewayResponse = new List<Allocation> { new Allocation() { Id = 1, PersonId = 2, AllocatedWorker = "Test Worker" } };
+
+            _mockDatabaseGateway.Setup(x => x.SelectAllocations(0, 0, null, 10))
+                .Returns(gatewayResponse);
+
+            var response = _allocationsUseCase.Execute(request);
+
+            response.Allocations.Should().BeEquivalentTo(gatewayResponse);
+        }
+
+        [Test]
+        public void ListAllocationsByTeamIdReturnsNothingIfNoMatches()
+        {
+            var request = new ListAllocationsRequest() { TeamId = 8 };
+            var gatewayResponse = new List<Allocation> { new Allocation() { Id = 1, PersonId = 2, AllocatedWorker = "Test Worker" } };
+
+            _mockDatabaseGateway.Setup(x => x.SelectAllocations(0, 0, null, 10))
```

Comment:
You could call execute with any request and it would return empty list ( that's what you are trying to test ) since you have no entires in the DB.

Reply:
Done on [d50eb72](https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/610/commits/d50eb727ec51d87e296be9b85f1392ebfb7118f4)